### PR TITLE
Generate AfidSet for testing

### PR DESCRIPTION
**Pull request by kaitj**
**Proposed changes**
This PR updates and renames the `afid_coord` strategy to `afid_set`, generating a near complete `AfidSet` object (_sans desc_). This allows for future testing of methods requiring a (in)valid `AfidSet`, while still being able to use a list of `AfidPositions` by invoking `AfidSet.afids`.

Additionally, this adds in an additional `AfidSet.load` to the `valid_save_fcsv` test, to ensure that the generated AfidSet that is saved can also be loaded.

**Types of changes**
What types of changes does your code introduce? Put an `x` in the boxes that apply

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other (if none of the other choices apply)

**Checklist**
Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you are unsure about any of the choices, don't hesitate to ask!

- [x] Changes have been tested to ensure that fix is effective or that a feature works.
- [x] Changes pass the unit tests
- [x] Code has been run through the `poe quality` task
- [x] I have included necessary documentation or comments (as necessary)
- [x] Any dependent changes have been merged and published

**Notes**
All PRs will undergo the unit testing before being reviewed. You may be requested to explain or make additional changes before the PR is accepted.

### CODE REVIEW

File: afids_utils/tests/strategies.py [hunk 1]
```diff
@@ -5,27 +5,29 @@
 from hypothesis.extra.numpy import arrays
 from numpy.typing import NDArray
 
-from afids_utils.afids import AfidPosition
+from afids_utils.afids import AfidPosition, AfidSet
 
 
 @st.composite
-def afid_coords(
+def afid_set(
```
Comment (tkkuehn) [inline afids_utils/tests/strategies.py:12]: By convention, `SearchStrategy`s should have a plural name, so this should be called `afid_sets`

File: afids_utils/afids.py [hunk 2]
```diff
@@ -19,7 +19,7 @@ class AfidPosition:
     x: float = attrs.field()
     y: float = attrs.field()
     z: float = attrs.field()
-    desc: str = attrs.field()
+    desc: str = attrs.field(default="")
```
Comment (tkkuehn) [inline afids_utils/afids.py:22]: I'd rather not have a default value that's invalid here (even if you want to leave the invalid `desc`s in the test for now)

File: afids_utils/tests/test_ext.py [hunk 3]
```diff
@@ -164,32 +164,29 @@ def test_valid_get_afids(self, valid_fcsv_file: PathLike[str], label: int):
 
 
 class TestSaveFcsv:
-    @given(afids_coords=afid_coords())
+    @given(afids_set=afid_set())
```
Comment (tkkuehn) [inline afids_utils/tests/test_ext.py:167]: Nitpick: I'd rather call this think `afid_set` for consistency with the class name

File: afids_utils/tests/test_ext.py [hunk 4]
```diff
@@ -211,17 +208,21 @@ def test_save_fcsv_valid_template(
...
+                    str(afids_set.afids[idx].x),
+                    str(afids_set.afids[idx].y),
+                    str(afids_set.afids[idx].z),
                 )
 
-    @given(afids_coords=afid_coords(bad_range=True))
-    def test_invalid_num_afids(self, afids_coords: list[AfidPosition]) -> None:
+            # Check to see if file can be loaded with afids-utils
+            test_load = AfidSet.load(out_fcsv_file.name)
+            assert isinstance(test_load, AfidSet)
```
Comment (tkkuehn) [inline afids_utils/tests/test_ext.py:218]: I _think_ we could even `assert test_load == afids_set`

**Review by tkkuehn — CHANGES_REQUESTED** [review]
A couple of minor comments but this largely looks good

**Comment by kaitj** [comment]
Ahh oops, this one was accidental, will see if I can roll back, otherwise will throw a patch in

File: afids_utils/afids.py (hunk 2 above)
Comment (kaitj) [reply afids_utils/afids.py:22]: Fair enough, I think alternatively I can load in the valid `descs` and any invalid ones set to "Unknown".

File: afids_utils/tests/test_ext.py (hunk 4 above)
Comment (kaitj) [reply afids_utils/tests/test_ext.py:218]: Ahh yeah, this test can be simplified quite a bit with the new `afid_sets` strategy

**Pull request merged**